lib/gossip/index.js

```diff
@@ -94,7 +94,6 @@ Gossip.prototype.start = function start() {
         return;
     }
 
-    this.ringpop.membership.shuffle();
```

Comment:
Now that the membership iterator is doing this lazy shuffle one of the characteristics of SWIM as described in `4.3. Round-Robin Probe Target Selection: Providing
Time-Bounded Strong Completeness` is not applicable any more.

In the paper they say "a newly joining member is inserted
in the membership list at a position that is chosen uniformly
at random". Now a newly joined member will only have a chance of being pinged when 1 node reaches the end of its pingable members and reshuffles the membership list into the members array of the MembershipIterator.